### DISCUSSION TITLE
Print warning when using intvar or boolvar in linear solver model builder with linear backends like glop or pdlp

### DISCUSSION THREAD

**Discussion by arnabadas** [google/or-tools#3696] [Feature requests]
**What language and solver does this apply to?**
Python (maybe other languages also, not sure), Linear Solver Model Builder

**Describe the problem you are trying to solve.**
Mistakenly using glop backend for mip problems should be accompanied by a warning.

**Describe the solution you'd like**
Warning saying that int vars and boolvars are handled as continuous variables when using linear programming backends like pdlp, glop.

**Describe alternatives you've considered**
print warning manually based on the solver name considered.

**Additional context**
NA

**Comment by lperron** [comment]
The issue with that is (1) warnings are spammy for apps in production, and (2) the reverse, most of the time, they are not printed out.
